packages/bar/src/BarItem.js

```diff
@@ -98,7 +98,7 @@ BarItem.propTypes = {
     borderWidth: PropTypes.number.isRequired,
     borderColor: PropTypes.string.isRequired,
 
-    label: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.number, PropTypes.element]).isRequired,
```

Comment:
This could be replaced by `PropTypes.node.isRequired`.

Reply:
You're right. I'll do the change.